setup.py

```diff
@@ -26,20 +26,20 @@
     platforms=['Any'],
     packages=['scrapinghub', 'scrapinghub.client', 'scrapinghub.hubstorage'],
     package_data={'scrapinghub': ['VERSION']},
-    install_requires=['requests>=1.0', 'retrying>=1.3.3', 'six>=1.10.0'],
-    extras_require={'msgpack': mpack_required},
+    install_requires= mpack_required + [
+        'requests>=1.0',
+        'retrying>=1.3.3',
+        'six>=1.10.0'
+    ],
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
```

Comment:
3.6 end of life: 23 Dec 2021. So I think we should either keep it or hold this change until that date.